hack/lib/serverless.bash

```diff
@@ -151,6 +153,32 @@ EOF
   logger.success 'Knative Eventing has been installed sucessfully.'
 }
 
+function deploy_knativekafka_cr {
+  logger.info 'Deploy Knative Kafka'
+
+  # Wait for the CRD to appear
+  timeout 900 "[[ \$(oc get crd | grep -c knativekafkas.operator.serverless.openshift.io) -eq 0 ]]" || return 6
+
+  # Install Knative Kafka
+  cat <<EOF | oc apply -f - || return $?
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  name: knative-kafka
+  namespace: ${EVENTING_NAMESPACE}
+spec:
+  source:
+    enabled: true
+  channel:
+    enabled: true
+    bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
```

Comment:
we have Kafka, do we?

Reply:
Yeah, installed  here: https://github.com/openshift-knative/serverless-operator/pull/520/files#diff-196879e6787eb2045a7c3ca7eac896b28577a5b8cea9eeae66b835a145616220R36